imap_data_access/__init__.py

```diff
@@ -73,6 +73,6 @@
 VALID_FILE_EXTENSION = {"pkts", "cdf"}
```

Comment:
can we update instruments name too?
```
VALID_INSTRUMENTS = {
    "codice",
    "glows",
    "hit",
    "hi",
    "idex",
    "lo",
    "mag",
    "swapi",
    "swe",
    "ultra",
}
```

Reply:
Also, is there a command missing between "l1cb" and "l1d"?

Reply:
Good catch on the instruments, updated! I also updated the l1cb / l1d, which I think was a typo from before.